plugins/inputs/cisco_telemetry_mdt_netconf/sample.conf

```diff
@@ -0,0 +1,88 @@
+
+  ## Redial interval if the client fails to connect to the server
+  ## Optional, default value: 10s.
+  # redial = "10s"
+
+  ## Telemetry streaming
+  ## IOS-XE Subscription - periodic
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.subscription]]
+    xpath_filter = "/if:interfaces-state/interface"
+    update_trigger = "periodic"
+    period = "10s"
+
+    ## Leaves to be marked as keys in Influx LINE format.
+    ## They are valid throughout all other defined operations.
+    keys = ["/if:interfaces-state/interface/name"]
+
+  ## IOS-XE Subscription - periodic
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.subscription]]
+    xpath_filter = "/mdt-oper:mdt-oper-data/mdt-subscriptions"
+    update_trigger = "periodic"
+    period = "10s"
+    keys = ["/mdt-oper:mdt-oper-data/mdt-subscriptions/subscription-id"]
+
+  ## IOS-XE Subscription - periodic
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.subscription]]
+    xpath_filter = "/memory-ios-xe-oper:memory-statistics/memory-ios-xe-oper:memory-statistic"
+    update_trigger = "periodic"
+    period = "10s"
+    keys = ["/memory-ios-xe-oper:memory-statistics/memory-ios-xe-oper:memory-statistic/memory-ios-xe-oper:name"]
+
+  ## IOS-XE Subscription - Xpath union for multiple subtrees
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.subscription]]
+    xpath_filter = "/interfaces-ios-xe-oper:interfaces/interface/statistics/in-octets|/interfaces-ios-xe-oper:interfaces/interface/statistics/out-octets"
+    update_trigger = "periodic"
+    period = "5s"
+    keys = ["/interfaces-ios-xe-oper:interfaces/interface/name"]
+
+  ## IOS-XE Subscription - on-change
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.subscription]]
+    xpath_filter = "/cdp-ios-xe-oper:cdp-neighbor-details/cdp-neighbor-detail"
+    update_trigger = "on-change"
+    period = "0s"
+
+  ## Get operations
+  ## IOS-XE Get Request with filter
+  [[inputs.cisco_telemetry_mdt_netconf.get_service.get]]
+    xpath_filter = "/interfaces-state/interface[name='GigabitEthernet1']/oper-status"
+    period = "10s"
+    keys = ["/interfaces-state/interface/name"]
+
+  ## IOS-XE Get Request with filter and multiple keys
+  [[inputs.cisco_telemetry_mdt_netconf.get_service.get]]
+    xpath_filter = "/interfaces-state/interface[name='GigabitEthernet2']"
+    period = "10s"
+    keys = ["/interfaces-state/interface/name", "/interfaces-state/interface/if-index"]
+
+  ## IOS-XE Get Request without filter
+  [[inputs.cisco_telemetry_mdt_netconf.get_service.get]]
+    xpath_filter = "/memory-statistics/memory-statistic"
+    period = "10s"
+    keys = ["/memory-statistics/memory-statistic/name"]
+
+  ## Event notification subscription
+  ## NSO Event notfication subscription with a key
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.notification]]
+    stream = "ncs-alarms"
+    keys = ["alarm-notification/alarm-class"]
```

Comment:
No newline at the end of the file.

(I don't know if this is needed, but it looks better)

Reply:
Addressed by @NetworkGuy-0x06